test/compare_on_data.cpp

```diff
@@ -212,16 +212,11 @@ void load_eos_spiner(const std::string &filename, const std::vector<int> &matids
                      std::vector<EOS> &eos_h, std::vector<EOS> &eos_d) {
   eos_h.clear();
   eos_d.clear();
-  EOSBuilder::EOSType type = EOSBuilder::EOSType::SpinerEOSDependsRhoT;
-  EOSBuilder::params_t params;
-  params["filename"] = filename;
-  params["reproducibility_mode"] = false;
 
   for (int matid : matids) {
     if (matid > 0) { // non-analytic
-      params["matid"] = matid;
-      eos_h.push_back(EOSBuilder::buildEOS(type, params));
-      eos_d.push_back(eos_h.back().GetOnDevice());
+      eos_h.push_back(SpinerEOSDependsRhoT(filename, matid))
```

Comment:
Missing `;`?

Reply:
Good catch. fixed.

Reply:
is this test not run in our standard CI testing? 

Has this been run on gitlab CI?

Reply:
It's not run our built (or even tied into the cmake). It's one of my old testing scripts from the early days of singularity. It's designed to read in something like an xrage dump and compare eospac vs spiner on realistic datasets. I don't want to delete it, as I think it will come in handy again at some point, but it's not something we need to worry too much about maintaining.

Reply:
Maybe we should have something like a deprecated/one-off/old directory to move some of this legacy stuff so as not to be confused with files in the test dir that are connected to cmake and commonly built/tested?

Reply:
Maybe that movement could/should be a separate PR, if so, an issue should be made.